src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -685,6 +679,14 @@ theorem getMsbD_setWidth {m : Nat} {x : BitVec n} {i : Nat} :
     · simp [h']
       omega
 
+-- This is a simp lemma as there is only a runtime difference between `setWidth'` and `setWidth`,
+-- and for verification purposes they are equivalent.
+theorem setWidth'_eq {x : BitVec w} (h : w ≤ v) : x.setWidth' h = x.setWidth v := by
```

Comment:
The comment confuses me. It says `This is a simp lemma`, but `setWidth'_eq` is not tagged simp. is this intentional?

Reply:
No, it's a mistake, thanks for catching it!